site/content/integrate/marketplace-submissions/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contribute to the Marketplace"
+heading: "Contribute to the Marketplace"
+description: "If you’ve built an integration, playbook, or tool which extends Mattermost functionality, consider sharing it with the community on the Mattermost Web Marketplace."
+weight: 100
+aliases: 
+  - /extend/plugins/community-plugin-marketplace/s
+---
+
+The {{<newtabref title="Mattermost Web Marketplace" href="https://mattermost.com/marketplace/">}} supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
```

Comment:
```suggestion
The {{<newtabref title="Mattermost Web Marketplace" href="https://mattermost.com/marketplace/">}} supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the Web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
```

Reply:
I think that instead of changing capitalizations on these documents for now, we should wait until their is an official style guide document. cc @cwarnermm @justinegeffen

Reply:
Docs has provided guidance on capitalization for the Mattermost Marketplace (see above). It's agreed that we need an official style guide document.